files/verify-cn.py

```diff
@@ -19,94 +20,152 @@
 DENY_USER_CONNECTION_ATTEMPT = 1
 
 # Configuration constants
+CONFIG_FILE = "verify-cn.yml"
 KEYTAB_FILE = "/etc/krb5.keytab"
-LOG_LEVEL = "INFO"
+PEER_CERT_VARIABLE = "peer_cert"
 
 
-def kinit():
+def load_client_certificate() -> Optional[str]:
+    """Read certificate data from file identified by environment variable."""
+    # OpenVPN sends us the client's certificate though the peer_cert environment
+    # variable.
+    try:
+        client_certificate_path: Path = Path(os.environ[PEER_CERT_VARIABLE])
+    except KeyError:
+        logging.critical(
+            "OpenVPN did not set the 'peer_cert' environement variable."
```

Comment:
Typo: `environement`

Reply:
Fixed in https://github.com/cisagov/ansible-role-openvpn/pull/18/commits/79c9a9199c6ee6436362e3181db86c6c46377c22 👍